singleapplication_p.cpp

```diff
@@ -146,15 +146,17 @@ void SingleApplicationPrivate::genBlockServerName()
     if( ! (options & SingleApplication::Mode::ExcludeAppPath) ){
 #ifdef Q_OS_WIN
```

Comment:
Maybe use `if defined` as in else block for consistency?

```suggestion
#if defined(Q_OS_WIN)
```